README.md

```diff
@@ -60,6 +60,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
   - [Programming Courses Free Online](#programming-courses-free-online)
+- [Tools](#tools)
```

Comment:
Add indentation for it to be placed under Others

Reply:
Thats it fixed. Misunderstood what you meant with under Others. Sorry about that 😄